src/notebook/output-area/model.ts

```diff
@@ -177,6 +177,9 @@ class OutputAreaModel implements IDisposable {
           model.output_type = msgType;
           this.add(model);
           break;
+        case 'clear_output':
+          this.clear(true);
```

Comment:
The argument should be the value given in the clear_output message: http://jupyter-client.readthedocs.io/en/latest/messaging.html#clear-output